setup.py

```diff
@@ -1,5 +1,12 @@
+# Checking python version: If python 3 was not found then it returns 1 and does
+#   not do anything
+from platform import python_version
+primer_version = python_version().split(".")
+if int(primer_version[0]) != 3:
+    print ("This madule only works with python3")
+    print ("To setup the madule simply run `python3 setup.py install`")
```

Comment:
madule->module
i'm surprised this doesn't conflict with what I just merged from Mahsa, I though the added this check.

Reply:
I am still confused as I think Mahsa and I have done something redundant (as I pointed out before her changes.) The latency also results of that. I'll send the pull request very soon.